templates/features/features_layout.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'base_layout.html' %}
+
+{% block content %}
+<section class="p-strip--main">
```

Comment:
p-strip--main is that a new local strip?

Reply:
The content is very close to the top. It could do with p-strip--suru--topped

Reply:
@anthonydillon I've added p-strip--suru-topped to the high-availability page - should I also add it to the docs page for consistency? 

high-availability: 

![image](https://user-images.githubusercontent.com/58959073/95464723-3250a600-0972-11eb-8edc-8eaec53f7eaf.png)


docs: 

![image](https://user-images.githubusercontent.com/58959073/95464758-3c72a480-0972-11eb-96eb-1e9627f9953f.png)